### DISCUSSION TITLE
Programs are not up to date in Winget repository.

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/winget-cli#745] [Q&A] [answered]
I come across lots of programs that are not up to date, what is being done about this?

as an example that happened just now, I wanted to install OBS Studio through Winget but the version it shows is too old.
25 vs 26.1.1 which is the newest and available on their website and Github.

![{E0917C3D-7D44-47F9-821E-396CF2A5218A}](https://user-images.githubusercontent.com/72892766/107156413-66c18680-6993-11eb-9509-e35afc3445cd.png)

**Comment by ItzLevvie** [comment, marked as answer]
OBS Studio hasn't been updated for a while because the newer versions of OBS Studio are blocked by a known issue from being added into the winget-pkgs repository; you can see the status of the PR over here: https://github.com/microsoft/winget-pkgs/pull/6463.

**Reply by ghost** [reply]
Thank you, i remember i saw Winrar had also an old version in Winget repository for a while but i checked and it's up to date now.

**Comment by Swe-it** [comment]
FortiClient VPN is also very out of date, winget offers 6.2 where Fortinet has a 7.2 version.
